samples/net/download/prj.conf

```diff
@@ -15,3 +15,7 @@ CONFIG_NET_IPV4=y
 CONFIG_HEAP_MEM_POOL_SIZE=1024
 
 CONFIG_NET_CONNECTION_MANAGER=y
+
+# Wi-Fi credentials
+CONFIG_WIFI_CREDENTIALS_STATIC_SSID=""
```

Comment:
I vote for removing these from `prj.conf` and rather depend on the documentation and the build assert that will appear if they are not set.

Having them in prj.conf doesn't quite makes sense due to the samples being invalid for LTE builds which (i belive?) will cause build warning when building for 91.

Reply:
Agreed, I don't think we can have them in prj.conf for that reason. Another option is to promote using the shell for credential provisioning

Reply:
I am fine with removing them, I mostly want to achieve a consensus on how it should be done for all samples which is supporting wifi. 

I did not, however, discover any build warnings when building a sample with this config on for 9161dk.

Reply:
@jtguggedal But do you think we should promote using the shell for credential provisioning in all samples or just one/a few?

Reply:
I think it could make sense to do it in all, just to be consistent. And we shouldn't really promote compiling the credentials into the firmware
What do you think @krish2718 and @simensrostad ?

Reply:
Yes, I would agree, at the very least we can put a warning that do not do this in the production.